components/common/Icon.js

```diff
@@ -19,6 +19,15 @@ const IconTag = s.img`
     margin-right: 3px;
     margin-top: -5px;
   }
+
+  .button.is-large & {
```

Comment:
Why is there an & after `.is-large`?

Reply:
We want to apply these rules to the icon, not the parent button. The icon is nested inside the parent button, it is not the button.